src/components/Rapidoc.js

```diff
@@ -3,8 +3,8 @@ import ExecutionEnvironment from "@docusaurus/ExecutionEnvironment";
 import Models from "../../docs/console-api/models/index";
 import { useColorMode } from '@docusaurus/theme-common';
 
-// In the order they will appear on All Endpoint's tab
-const supportedEntities = ["gates", "experiments", "dynamic-configs", "segments", "autotunes", "holdouts" ]
+// In the order they will appear on All Endpoint's tab preferably
+const supportedEntities = ["gates", "experiments", "layers", "dynamic-configs", "segments", "autotunes", "holdouts" ]
```

Comment:
Layers is pretty advanced...I think I'd order these more like:

gates, segments, dynamic configs, experiments, holdouts, layers, autotunes

Reply:
[8ca6cdb](https://github.com/statsig-io/docs/pull/346/commits/8ca6cdb02a42d72dd3a5d10a40a8e87c673fe7f2)